ScoutSuite/core/server.py

```diff
@@ -0,0 +1,107 @@
+from sqlitedict import SqliteDict
+import cherrypy
+import cherrypy_cors
+
+import re
+
+count_re = re.compile(r".*_count$")
+
+class Server(object):
+    def __init__(self, filename):
+        self.results = SqliteDict(filename)
+
+    @cherrypy.expose()
+    @cherrypy.tools.json_out()
+    def summary(self):
```

Comment:
This is a bit cryptic, can we add a docstring? Unit tests would also be appropriate.